### DISCUSSION TITLE
Question about Prisma & GraphQL auto-generated schema files

### DISCUSSION THREAD

**Discussion by NHDang95** [keystonejs/keystone#7293] [Questions] [answered]
Hello! I've just started learning KeystoneJS. As the title suggests, can anyone tell to me how KeystoneJS version or any dependencies version affect those 2 generated files' content?

**Comment by raveling** [comment, marked as answer]
The version of Keystone will affect the auto generated files you get. 

Is there anything more specific you would like to know?
